src/WebdriverClassicDriver.php

```diff
@@ -1386,5 +1389,10 @@ private function jsonEncode($value, string $action, string $field): string
         }
     }
 
+    private function wrapError(Throwable $e): Exception
+    {
+        return $e instanceof Exception ? $e : new ErrorException($e);
```

Comment:
I think we should rather add support for any Throwable as the previous exception in Mink's DriverException instead of having this wrapping (after all, the only thing it does with it is passing it to the parent constructor).

Reply:
I think that should be doable without requiring a version since it concerns constructors, right?

If so, could you please take care of it?

Reply:
See https://github.com/minkphp/Mink/pull/855